### PR TITLE
Add blog post about scalameta and scalafix release

### PR DESCRIPTION
**Pull request by olafurpg**

### CODE REVIEW

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+---
+category: blog
+by: Ólafur Páll Geirsson, Eugene Burmako
+title: "Build code analyzers with Scalameta and Scalafix"
+---
+
+We are excited to announce the release of
+[Scalameta v4.0.0](https://github.com/scalameta/scalameta/releases/tag/v4.0.0)
+and
+[Scalafix v0.9.0](https://github.com/scalacenter/scalafix/releases/tag/v0.8.0-RC1)
```
Comment (Duhemm) [inline _posts/2018-11-16-scalafix-scalameta.md]: This points to v0.8.0-RC1

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+A big focus of this release has been documentation. Scalameta and Scalafix have
+new websites with guides, cookbooks and reference documents.
+
+Visit [scalameta.org](https://scalameta.org/) to see the new Scalameta
+documentation. Key pages of the site include:
+
+- [SemanticDB specification](https://scalameta.org/docs/semanticdb/specification.html):
+  learn the details of the new semantic API.
+- [SemanticDB guide](https://scalameta.org/docs/semanticdb/specification.html):
```
Comment (Duhemm) [inline _posts/2018-11-16-scalafix-scalameta.md]: This should probably point to https://scalameta.org/docs/semanticdb/guide.html

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  `metap`.
+- [Trees guide](https://scalameta.org/docs/trees/guide.html): learn how to
+  parse, construct, traverse and transform Scala syntax trees.
+- [ScalaFiddle playground](https://scalameta.org/docs/trees/scalafiddle.html):
+  explore Scalameta APIs directly in the browser with access to code completions
+  and fast edit/compile/run feedback. Big thanks to
+  [scalafiddle.io](https://scalafiddle.io/) for providing a great service!
+
+Visit
+[scalacenter.scalafix.io/scalafix](https://scalacenter.scalafix.io/scalafix/) to
```
Comment (Duhemm) [inline _posts/2018-11-16-scalafix-scalameta.md]: `scalacenter.github.io` instead?

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+see the new Scalafix documentation. Key pages of the site include:
+
+- [Implementing custom rules tutorial](https://scalacenter.github.io/scalafix/docs/developers/tutorial.html):
+  learn all the steps of implementing a custom rewrite or linter rule, starting
+  from tests and ending with publishing.
+- [Patch reference](https://scalacenter.github.io/scalafix/docs/developers/tutorial.html):
```
Comment (Duhemm) [inline _posts/2018-11-16-scalafix-scalameta.md]: Should probably be: https://scalacenter.github.io/scalafix/docs/developers/patch.html

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+## Share your code analyzer
+
+A new functionality in this release is the ability to easily install and run
+custom Scalafix rules. For sbt users, custom rules can be installed with the
+`scalafixDependencies` setting and discovered from the sbt shell via tab
+completions. For rule authors, a Scalafix rule is published to Maven Central as
+an ordinary library.
+
+Within a day after the Scalafix release,
+[Eugene Yakota](https://github.com/eed3si9n/) had already published a Scalafix
```
Comment (SethTisue) [inline _posts/2018-11-16-scalafix-scalameta.md]: s/Yakota/Yokota/

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+and
+[Scalafix v0.9.0](https://github.com/scalacenter/scalafix/releases/tag/v0.8.0-RC1)
+introducing new APIs enabling more advanced source code analysis. Scalameta is a
+library to read, analyze, transform and generate Scala programs. Scalafix is a
+refactoring and linting tool.
+
+This release is the result of a close collaboration between the Scala Center and
+Twitter along with contributions from over 30 contributors in the community. Big
+thanks to everybody who made this happen!
+
```
Comment (SethTisue) [inline _posts/2018-11-16-scalafix-scalameta.md]: you might consider adding a tl;dr at this point, some bullets that say what the rest of the blog post is going to contain

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+Synthetics were contributed by [Max Ovsiankin](http://github.com/maxov) during
+his internship at Twitter this summer. Max did a great job and synthetics
+represent only a fraction of his contributions this summer.
+
+## Share your code analyzer
+
+A new functionality in this release is the ability to easily install and run
+custom Scalafix rules. For sbt users, custom rules can be installed with the
+`scalafixDependencies` setting and discovered from the sbt shell via tab
```
Comment (SethTisue) [inline _posts/2018-11-16-scalafix-scalameta.md]: maybe show an example `build.sbt` snippet?

File: _posts/2018-11-16-scalafix-scalameta.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+completions. For rule authors, a Scalafix rule is published to Maven Central as
```
Comment (SethTisue) [inline _posts/2018-11-16-scalafix-scalameta.md]: maybe show an example of tab completion working?

File: _posts/2018-11-16-scalafix-scalameta.md (hunk 1 above)
Comment (SethTisue) [reply _posts/2018-11-16-scalafix-scalameta.md]: Huh? I don't understand. (And, everyone will think it's a mistake, regardless of what the explanation is.)

File: _posts/2018-11-16-scalafix-scalameta.md (hunk 8 above)
Comment (SethTisue) [reply _posts/2018-11-16-scalafix-scalameta.md]: cool!

File: _posts/2018-11-16-scalafix-scalameta.md (hunk 1 above)
Comment (olafurpg) [reply _posts/2018-11-16-scalafix-scalameta.md]: I copy pasted the v0.8.0-RC1 release notes to the v0.9.0 release to avoid the need to explain :)

**Pull request merged**